dev/.astro/test_dag_integrity_default.py

```diff
@@ -0,0 +1,130 @@
+"""Test the validity of all DAGs. **USED BY DEV PARSE COMMAND DO NOT EDIT**"""
```

Comment:
We probably should delete this if we're not using

Reply:
this also has some tests that may be helpful to catch errors before deploying